scripts/utils/update_package.py

```diff
@@ -283,13 +326,56 @@ def update_dynamic_url(package):
         return latest_version
 
 
+# Update package that url ends with .msixbundle
+def update_msixbundle_url(package):
+    install_script_path, content = get_install_script(package)
+
+    if not content:
+        return None
+
+    # Match urls like:
+    # - https://windbg.download.prss.microsoft.com/dbazure/prod/1-2502-25002-0/windbg.msixbundle
+    m = re.search(
+        r"[\"'](?P<url>https{0,1}:\/\/.+?[A-Za-z\-_]+\/(?P<version>\d+\-\d+\-\d+\-\d+)\/[A-Za-z\-_]+\.msixbundle)[\"']",
+        content,
+    )
+    if not m:
+        return None
+
+    version = m.group("version")
+
+    sha256_rex = re.search(r"[\"'](?P<sha256>[A-Fa-f0-9]{64})[\"']", content)
```

Comment:
[nitpick] you renamed the variable `rex` to `m` above for consistency with the rest of the script. Can you rename this one too? :smile: 

```suggestion
    sha256_m = re.search(r"[\"'](?P<sha256>[A-Fa-f0-9]{64})[\"']", content)
```